recipe/meta.yaml

```diff
@@ -10,24 +10,22 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c93fc0ed8216c71f9b8264788ca7743e26f4498b2011e5e2404696e5fd238e68
+  sha256: c8e9d769a6066bb5f94bb0362dbc20ed6f6b525336373263aeb6f9c6dc7b91e4
 
 build:
   number: 0
   skip: true  # [py<37 or osx]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}  # [win]
+    - {{ compiler('cxx') }}  # [win]
+    - {{ compiler('c') }} =11.2  # [not win]
+    - {{ compiler('cxx') }} =11.2  # [not win]
```

Comment:
nit: `# [not win]` -> `# [linux]`

Reply:
true enough, for a moment I thought we needed to worry about osx

Reply:
`# [unix]` would cover `osx` indeed, so I suggested `# [linux]` 🙂